yaksh/management/commands/add_moderator.py

```diff
@@ -0,0 +1,37 @@
+'''
+   This command adds users to the moderator group with permissions to add, change and delete
+   the objects in the exam app.
+'''
+
+# django imports
+from django.core.management.base import BaseCommand, CommandError
+from django.contrib.auth.models import User, Group, Permission
+from django.contrib.contenttypes.models import ContentType
+from django.db.utils import IntegrityError
+
+# Yaksh imports
+from yaksh.models import Profile
+
+class Command(BaseCommand):
+    help = 'Adds users to the moderator group'
+
+    def add_arguments(self, parser):
+        # Positional arguments
+        parser.add_argument('usernames', nargs='+', type=str)
+
+    def handle(self, *args, **options):
+        try:
+            group = Group.objects.get(name='moderator')
+        except Group.DoesNotExist:
```

Comment:
Why can't the group be created here? Is it a separate command? Why not make it a part of this?

Reply:
There is a separate command, it is preferable that both actions be kept individual.

Reply:
Why? I don't know why it is preferable to have to separate commands. Could you explain it?

Reply:
In the current situation:
- There is already a command that adds moderators, so it is not required to add the same functionality here.
- If you refer to combining the two commands and dropping the ```create_mod_group``` command then that fails in the current scenario because the users are mandatory positional arguments, if the deployer tries to add moderator group without having any users in the database this command will fail.

Reply:
I will change the argument syntax so that the users can be added optionally and a moderator group is added by default.

Reply:
I second that idea. Will this be added in this PR itself?